composer.json

```diff
@@ -5,25 +5,24 @@
     "license": "GPL 3.0",
     "type": "project",
     "require": {
-        "php": "^7.4",
+        "php": "^7.4|^8.0",
         "ext-json": "*",
         "ext-curl": "*",
         "ext-pdo": "*",
         "ext-simplexml": "*",
-        "guzzlehttp/guzzle": "^6.3",
+        "guzzlehttp/guzzle": "^7.0",
         "bugsnag/bugsnag-laravel": "^2.17",
         "predis/predis": "~1.0",
         "laravel/passport": "^10.0",
         "laravel/framework": "^8.0",
         "beyondcode/laravel-websockets": "^2.0",
-        "knplabs/github-api": "^2.12",
-        "php-http/guzzle6-adapter": "^1.1",
+        "knplabs/github-api": "^3.0",
         "nesbot/carbon": "^2.32",
         "maatwebsite/excel": "^3.1",
         "mjaschen/phpgeo": "^3.0"
     },
     "require-dev": {
-        "fzaninotto/faker": "~1.4",
+        "fakerphp/faker": "^1.13",
```

Comment:
Why the downgrade on min version?

Reply:
Old repo is being closed (ref: https://marmelab.com/blog/2020/10/21/sunsetting-faker.html) and replaced by https://github.com/FakerPHP/Faker. The new package doesn't have its releases/tags aligned to the old package thus the latest release is `v1.13.0`